src/main/webapp/WEB-INF/templates/index.html

```diff
@@ -2,30 +2,40 @@
 <html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
 <head>
     <!-- Required meta tags -->
-    <meta charset="utf-8">
+    <meta charset="UTF-8">
```

Comment:
index.html mi prosim nezmen vobec lebo budu konflikty. Resources potom doplnime

Reply:
Máš niečo rozrobené? Lebo toto sú doležité zmeny tie th:if bloky, inak nepôjdu css js a favicon na serveri, ale ako viem to dočasne dať preč, konfliktov som už v praxi niekoľko vyriešil tak neni problém ak sa len toho bojíš.

Reply:
proste som ho aj ja zmenil, len to som ti chcel zdelit :)